### PR TITLE
created new app `friends`

### PR DESCRIPTION
**Pull request by animeshrick**
add new app to settings.py
create models - friend and friendrequest
register models to admin
test adding & deleting from admin site

### Overview

> Github Issue:[ #4 ](https://github.com/KoushikMallik-developer/split-it-api-v2/issues/4])

**What was the problem:**
Friend models are not created.

**How was this solved:**
Creating new models using "GenericBaseModel" with attributes as mentioned in the issue. 
Tested adding/deleting all models using admin panel.

**Features:**
- created new app `friends`
- add new app to settings.py
- create models - friend and friendrequest
- register models to admin
- test adding & deleting from admin site

**Services to be impacted:**
- `split-it-api-v2`

**Example Code or Screenshot:**
```
Example Code or Screenshot
```

**_For the PR Creator_**

- [x] Filled out entirely of PR Template.
- [x] Changes meet acceptance criteria defined in ticket.
- [x] Unit Tests Passed. :tada:
- [x] Tested on DEV Env.
- [ ] Tested on QA Env.

### CODE REVIEW

File: friends/models.py
Comment (KoushikMallik-developer) [inline friends/models.py:1]: This file "models.py" is not needed, so remove it.

File: friends/models/friend_model.py
Comment (KoushikMallik-developer) [inline friends/models/friend_model.py:1]: The file name should be only "friend.py"

File: friends/models/friend_model.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from auth_api.models.base_models.base_model import GenericBaseModel
+from django.db import models
+
+
+class Friend(GenericBaseModel):
+    user1 = models.CharField(max_length=100)
+    user2 = models.CharField(max_length=100)
```
Comment (KoushikMallik-developer) [inline friends/models/friend_model.py]: The user1 and user2 should be one to many relationship.

File: friends/models/friend_request_model.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from auth_api.models.base_models.base_model import GenericBaseModel
+from django.db import models
+
+
+class FriendRequest(GenericBaseModel):
+    sender = models.CharField(max_length=100)
+    receiver = models.CharField(max_length=100)
```
Comment (KoushikMallik-developer) [inline friends/models/friend_request_model.py]: Same as Friend model, it should be one to many relationship.

File: requirements.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 annotated-types==0.7.0
 asgiref==3.8.1
+async-timeout==5.0.1
```
Comment (KoushikMallik-developer) [inline requirements.txt:3]: I am not seeing any usage of this library, any specific reason to add it? Else remove unnecessary requirements.

**Review by KoushikMallik-developer — CHANGES_REQUESTED** [review]

File: friends/models.py
Comment (animeshrick) [reply friends/models.py:1]: Removed

**Review by KoushikMallik-developer — APPROVED** [review]

**Pull request merged**